forms-flow-components/src/components/CustomComponents/FormTextArea.tsx

```diff
@@ -42,6 +45,9 @@ export const FormTextArea = forwardRef<HTMLTextAreaElement, FormTextAreaProps>((
     minRows = 1,
     onIconClick,
     maxRows = 5,
+    iconTop,
```

Comment:
Instead of passing multiple props, you can simplify it by passing a single iconPosition prop. This prop can accept values like CENTER, BOTTOM, or TOP. Based on the value of this prop, you can implement your logic accordingly. The value of the iconPosition prop should be treated case-insensitively to ensure flexibility.

Reply:
Agree . 
@fahad-aot Please add only 1 prop with values expecting 'top' , 'center' , 'bottom' . Keep 'top' as default.

Reply:
Updated.